build.assets/Makefile

```diff
@@ -315,7 +315,7 @@ release-windows: buildbox
 #
 .PHONY: docsbox
 docsbox:
-	docker pull $(DOCSBOX) || true
+	if ! docker inspect --type=image $(DOCSBOX) 2>&1 >/dev/null; then docker pull $(DOCSBOX) || true; fi
```

Comment:
`DOCSBOX` points to a mutable tag `quay.io/gravitational/next:main`
So if we update the docsbox image without adding a new tag, Drone won't pick up the changes?

If this is for testing changes in the docsbox image itself, using a different name seems more natural

Reply:
Drone uses Docker-in-Docker, so there's no image cache and this will always have an exit code > 0. It'll pull the image every time it runs.

Reply:
Ack. I suspect that this behavior in Drone will eventually change, but I guess it's fine for now.